driver/desktop/cursor.go

```diff
@@ -0,0 +1,24 @@
+package desktop
+
+// Cursor represents a standard fyne cursor
+type Cursor string
+
+const (
+	// DefaultCursor is the default cursor typically an arrow
+	DefaultCursor Cursor = "DefaultCursor"
```

Comment:
It just occurred to me also that this would probably be more efficient as an enum... using string comparison for an internal-only lookup seems like we're burning cycles for no benefit.